2021-05-30T14:07:32.555+0300	INFO	service command start completed	{"source": "main"}
2021-05-30T14:07:32.574+0300	INFO	starting kubemq sources connectors version: v0.7.0	{"source": "main"}
2021-05-30T14:08:24.656+0300	INFO	kubemq sources connectors stopped	{"source": "main"}
2021-05-30T14:08:24.662+0300	INFO	service command stop completed	{"source": "main"}
2021-05-30T14:09:01.339+0300	INFO	service command uninstall completed	{"source": "main"}
2021-05-30T14:12:35.277+0300	ERROR	Failed to uninstall kubemq-sources: service KubeMQ-Sources is not installed	{"source": "main"}
2021-05-30T14:12:42.561+0300	INFO	service command install completed	{"source": "main"}
2021-05-30T14:12:48.262+0300	INFO	starting kubemq sources connectors as a service version: v0.7.0	{"source": "main"}
2021-05-30T14:12:48.277+0300	INFO	1 bindings loaded	{"source": "config"}
2021-05-30T14:12:48.279+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:48.279+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:48.279+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:48.279+0300	ERROR	failed to initialized binding, folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:48.262+0300	INFO	service command start completed	{"source": "main"}
2021-05-30T14:12:49.289+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:49.290+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:49.290+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:49.290+0300	ERROR	failed to initialized binding: source-1, attempt: 1, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:50.304+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:50.304+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:50.304+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:50.305+0300	ERROR	failed to initialized binding: source-1, attempt: 2, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:51.316+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:51.317+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:51.317+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:51.318+0300	ERROR	failed to initialized binding: source-1, attempt: 3, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:52.331+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:52.332+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:52.332+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:52.333+0300	ERROR	failed to initialized binding: source-1, attempt: 4, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:53.346+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:53.347+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:53.347+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:53.347+0300	ERROR	failed to initialized binding: source-1, attempt: 5, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:54.363+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:54.363+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:54.364+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:54.364+0300	ERROR	failed to initialized binding: source-1, attempt: 6, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:55.376+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:55.376+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:55.376+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:55.376+0300	ERROR	failed to initialized binding: source-1, attempt: 7, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:56.389+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:56.389+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:56.390+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:56.390+0300	ERROR	failed to initialized binding: source-1, attempt: 8, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:57.405+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:57.405+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:57.406+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:57.406+0300	ERROR	failed to initialized binding: source-1, attempt: 9, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:58.406+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:58.406+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:58.407+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:58.407+0300	ERROR	failed to initialized binding: source-1, attempt: 10, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:12:59.419+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:12:59.419+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:12:59.419+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:12:59.419+0300	ERROR	failed to initialized binding: source-1, attempt: 11, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:00.433+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:00.433+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:00.434+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:00.434+0300	ERROR	failed to initialized binding: source-1, attempt: 12, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:01.445+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:01.445+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:01.445+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:01.445+0300	ERROR	failed to initialized binding: source-1, attempt: 13, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:02.461+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:02.461+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:02.461+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:02.461+0300	ERROR	failed to initialized binding: source-1, attempt: 14, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:03.463+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:03.463+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:03.463+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:03.463+0300	ERROR	failed to initialized binding: source-1, attempt: 15, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:04.464+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:04.464+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:04.465+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:04.465+0300	ERROR	failed to initialized binding: source-1, attempt: 16, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:05.479+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:05.479+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:05.479+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:05.480+0300	ERROR	failed to initialized binding: source-1, attempt: 17, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:06.493+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:06.493+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:06.493+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:06.494+0300	ERROR	failed to initialized binding: source-1, attempt: 18, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:07.502+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:07.502+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:07.502+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:07.502+0300	ERROR	failed to initialized binding: source-1, attempt: 19, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:08.512+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:08.512+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:08.512+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:08.513+0300	ERROR	failed to initialized binding: source-1, attempt: 20, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:09.527+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:09.527+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:09.528+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:09.528+0300	ERROR	failed to initialized binding: source-1, attempt: 21, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:10.536+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:10.536+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:10.537+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:10.537+0300	ERROR	failed to initialized binding: source-1, attempt: 22, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:11.549+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:11.549+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:11.549+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:11.549+0300	ERROR	failed to initialized binding: source-1, attempt: 23, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:12.559+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:12.560+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:12.560+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:12.560+0300	ERROR	failed to initialized binding: source-1, attempt: 24, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:13.571+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:13.571+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:13.571+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:13.571+0300	ERROR	failed to initialized binding: source-1, attempt: 25, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:14.572+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:14.572+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:14.572+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:14.572+0300	ERROR	failed to initialized binding: source-1, attempt: 26, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:15.586+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:15.586+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:15.586+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:15.587+0300	ERROR	failed to initialized binding: source-1, attempt: 27, error: folder C:\e\demo\s3 path does not exist	{"source": "binding-service"}
2021-05-30T14:13:15.956+0300	INFO	1 bindings loaded	{"source": "config"}
2021-05-30T14:13:15.956+0300	INFO	config file changed, reloading...	{"source": "config"}
2021-05-30T14:13:15.957+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:13:15.959+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:13:15.959+0300	INFO	1 bindings loaded	{"source": "config"}
2021-05-30T14:13:15.960+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:13:15.960+0300	INFO	binding: source-1, started successfully	{"source": "source-1"}
2021-05-30T14:13:31.062+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:31.063+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:31.063+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:32.066+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:32.067+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:32.067+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:33.079+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:33.080+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:33.080+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:34.080+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:34.080+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:34.080+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:35.082+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:36.824+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:36.825+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:37.104+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:37.105+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:37.105+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:38.119+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:38.119+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:38.119+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:39.119+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:39.120+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:39.120+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:40.126+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:40.126+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:40.126+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:41.134+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:41.135+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:41.135+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:42.136+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:42.136+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:42.137+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:43.150+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:45.153+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:45.153+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:46.154+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:46.154+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:46.154+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:47.157+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:47.158+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:47.158+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:48.163+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:48.163+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:48.163+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:49.178+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:49.178+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:49.178+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:50.191+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:50.191+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:50.191+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:51.191+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:51.191+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:51.191+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:52.203+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:52.203+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:52.203+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:53.216+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:53.217+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:53.217+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:54.232+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:54.233+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:54.233+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:55.244+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:57.537+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:57.537+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:58.274+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:58.274+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:58.274+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:59.289+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:13:59.289+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:13:59.289+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:00.291+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:00.291+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:00.291+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:01.298+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:01.298+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:01.298+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:02.300+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:02.301+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:02.301+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:03.304+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:03.304+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:03.305+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:04.310+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:04.310+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:04.310+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:05.325+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:05.326+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:05.326+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:06.329+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:06.329+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:06.329+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:07.341+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:07.341+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:07.342+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:08.347+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:08.347+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:08.347+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:09.359+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:09.359+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:09.359+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:10.362+0300	INFO	sending file file: e:\demo\s3\kubemq-sources.log, size: 15046 bytes started 	{"source": "source-1"}
2021-05-30T14:14:10.362+0300	ERROR	retry 0 failed, error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:10.362+0300	ERROR	error during sending file kubemq-sources.log, All attempts fail:
#1: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp [::1]:50000: connectex: No connection could be made because the target machine actively refused it."	{"source": "source-1"}
2021-05-30T14:14:11.362+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:12.367+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:13.378+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:14.391+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:15.403+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:16.407+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:17.409+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:18.423+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:19.438+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:20.448+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:21.455+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:22.458+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:23.469+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:24.471+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:25.484+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:26.487+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:27.492+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:28.494+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:29.509+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:30.521+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:31.534+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:32.538+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:33.550+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:34.565+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:35.565+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:36.571+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:37.581+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:38.589+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:39.594+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:40.609+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:41.623+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:42.639+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:43.654+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:44.662+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:45.676+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:46.690+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:47.691+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:48.702+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:49.703+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:50.718+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:51.730+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:52.744+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:53.748+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:54.762+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:55.776+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:56.776+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:57.791+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:58.806+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:14:59.806+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:00.809+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:01.822+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:02.833+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:03.837+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:04.841+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:05.854+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:06.867+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:07.868+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:08.872+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:09.886+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:10.895+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:11.911+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:12.924+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:13.927+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:14.927+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:15.938+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:16.939+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:17.941+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:18.955+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:19.965+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:20.973+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:21.974+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:22.984+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:23.987+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:24.995+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:26.005+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:27.017+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:28.027+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:29.042+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:30.053+0300	ERROR	error during creating file requests e:\demo\s3\kubemq-sources.log, open e:\demo\s3\kubemq-sources.log: The system cannot find the file specified.	{"source": "source-1"}
2021-05-30T14:15:30.669+0300	INFO	kubemq sources connectors stopped	{"source": "main"}
2021-05-30T14:15:30.752+0300	INFO	starting kubemq sources connectors as a service version: v0.7.0	{"source": "main"}
2021-05-30T14:15:30.766+0300	INFO	1 bindings loaded	{"source": "config"}
2021-05-30T14:15:30.767+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:15:30.769+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:15:30.769+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:15:30.769+0300	INFO	binding: source-1, started successfully	{"source": "source-1"}
2021-05-30T14:15:30.752+0300	INFO	service command restart completed	{"source": "main"}
2021-05-30T14:30:42.602+0300	INFO	1 bindings loaded	{"source": "config"}
2021-05-30T14:30:42.602+0300	INFO	config file changed, reloading...	{"source": "config"}
2021-05-30T14:30:42.603+0300	INFO	binding: source-1, stopped successfully	{"source": "source-1"}
2021-05-30T14:30:42.603+0300	INFO	binding: source-1 target: initialized successfully	{"source": "source-1"}
2021-05-30T14:30:42.603+0300	INFO	binding: source-1 source initialized successfully	{"source": "source-1"}
2021-05-30T14:30:42.603+0300	INFO	binding: source-1, initialized successfully	{"source": "source-1"}
2021-05-30T14:30:42.603+0300	INFO	binding: source-1, started successfully	{"source": "source-1"}
2021-05-30T14:30:42.604+0300	INFO	1 bindings loaded	{"source": "config"}
2021-05-30T14:39:52.000+0300	INFO	service command uninstall completed	{"source": "main"}
2021-05-30T14:43:42.113+0300	INFO	kubemq sources connectors stopped	{"source": "main"}
2021-05-30T15:31:01.259+0300	INFO	generated manifest.json completed	{"source": "kubemq-sources"}
